lighthouse-core/report/templates/report-template.html

```diff
@@ -189,6 +189,8 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   <div class="subitem-result">
                     {{#if subItem.comingSoon}}
                           <span class="subitem-result__unknown score-unknown-bg">N/A</span>
+                    {{else if subItem.error}}
+                      <span class="subitem-result__unknown score-error-bg">N/A</span>
```

Comment:
this ends up looking like

<img width="783" alt="screen shot 2017-02-01 at 15 31 48" src="https://cloud.githubusercontent.com/assets/316891/22531053/93d7094a-e893-11e6-853c-ff509fe4d951.png">